Cargo.toml

```diff
@@ -33,8 +33,10 @@ serde = { version = "1.0", default-features = false, optional = true }
 serde_arrays = { version = "0.1.0", optional = true }
 hex = { version = "0.4", optional = true, default-features = false, features = ["alloc", "serde"] }
 blake2b_simd = "1"
+bls12_381 = { git = "https://github.com/privacy-scaling-explorations/bls12_381", branch =  "tmp_curve_ext", features = ["groups", "basefield"] }
```

Comment:
Hey Carlos, depending on a branch would be problematic here.

Every time people install Halo2curve or the CI runs the test, they pull the branch head. So if an incompatible commit is introduced to `tmp_curve_ext` branch then all the packages break. 

We should release a tag on bls12_381 repo and then depends on the tag here.

Reply:
I'm happy to release a tag and update it here as well as rebasing this!
Will do it within the next days! :smile:

Reply:
Indeed the release was done already back in October. See: https://github.com/privacy-scaling-explorations/bls12_381/releases/tag/v2023_10_26

Will update this in `main`

Reply:
Updated in [9329af1](https://github.com/privacy-scaling-explorations/halo2curves/pull/87/commits/9329af1e42fae4c3feb09c17a0a6ebcd7e2e6813)